tests/test_selenium.py

```diff
@@ -98,7 +101,7 @@ def test_add_to_cart(self):
         ActionChains(self.driver).move_to_element(first_dress).perform()
         assert_that(*self.add_to_cart, equal_to_ignoring_case("Add to cart"))
         self.driver.find_element(*self.add_to_cart).click()
-        self.driver.implicitly_wait(3)
+        self.driver.implicitly_wait(2)
         self.driver.find_element(*self.close_pop_up).click()
```

Comment:
Rafał, we have discussed that 'implicit wait jest ustawiany tylko 1 raz na wszyscy testy'. Why you have self.driver.implicitly_wait(2) in each test? It should be set up only ones in setup() method